src/modules/esl-related-target/test/related-target.test.ts

```diff
@@ -0,0 +1,44 @@
+import {ESLRelatedTarget} from '../core/esl-related-target';
+import {ESLToggleable} from '../../esl-toggleable/core/esl-toggleable';
+
+describe('ESLRelatedTarget: show/hide depending on mixin element state', () => {
+  beforeAll(() => ESLToggleable.register());
+  beforeAll(() => ESLRelatedTarget.register());
+
+  const $el = document.createElement(ESLToggleable.is) as ESLToggleable;
+  const $relatedEl = document.createElement(ESLToggleable.is) as ESLToggleable;
+  $relatedEl.setAttribute('id', 'related-toggleable');
+  $el.setAttribute(ESLRelatedTarget.is, '#related-toggleable');
+
+  jest.useFakeTimers();
+  beforeAll(() => document.body.appendChild($el));
+  beforeAll(() => document.body.appendChild($relatedEl));
+  afterAll(() => ($el.parentElement === document.body) && document.body.removeChild($el));
+  afterAll(() => ($relatedEl.parentElement === document.body) && document.body.removeChild($relatedEl));
+
+  describe('Synchronization of mixin and target states: check SHOW_REQUEST event', () => {
+    test('SHOW_REQUEST on the current mixin element leads to show target element', () => {
+      $el.hide();
+      jest.advanceTimersByTime(1);
+      expect($el.open).toBe(false);
+      expect($relatedEl.open).toBe(false);
+      $el.dispatchEvent(new CustomEvent($el.SHOW_REQUEST_EVENT));
+      jest.advanceTimersByTime(1);
+      expect($el.open).toBe(true);
+      expect($relatedEl.open).toBe(true);
+    });
```

Comment:
This test checks two expectations so I think it should be split into 2 different tests. 
By default Jest runs all the tests serially in the order they were encountered in the collection phase, waiting for each to finish and be tidied up before moving on. So just split this test into 2 tests.

Reply:
I'll split it)